src/com/google/javascript/jscomp/PolymerPass.java

```diff
@@ -171,9 +183,16 @@ public static boolean isPolymerCall(Node value) {
     return value != null && value.isCall() && value.getFirstChild().matchesQualifiedName("Polymer");
   }
 
-  /**
-   * Any member of a Polymer element or Behavior. These can be functions, properties, etc.
-   */
+  /** @return Whether the class extends Polymer.Element */
+  @VisibleForTesting
+  public static boolean isPolymerClass(Node value) {
+    return value != null
+        && value.isClass()
+        && !value.getSecondChild().isEmpty()
+        && value.getSecondChild().matchesQualifiedName("Polymer.Element");
```

Comment:
Yes. I actually have the next large chunk of work done which includes this, but I am trying to limit the size of the PRs.